### PR TITLE
Implement Restaurants page

### LINKED ISSUE #8: Implement Restaurants page
Implement design using antd `Card` component https://ant.design/components/card#components-card-demo-flexible-content

It should be rendered as `List` https://ant.design/components/list.

Screenshot 2023-10-06 at 15 49 59

You can use images and names from https://guide.michelin.com/en/restaurants

### PR DESCRIPTION
**Pull request by adam-lzk**
Closes #8 

Couldn't implement a field with a restaurant's price range, cuisine and rating: Card component has only one description field that was occupied by another text

### CODE REVIEW

File: src/pages/Restaurants.jsx [hunk 1]
```diff
@@ -1,7 +1,96 @@
...
+
+const Restaurants = () => (
+  <StyledContent>
+    <List
+      grid={{
+      gutter: 0,
+      column: 3,
+    }}
+      dataSource={cardsData}
+      renderItem={(item) => (
```
Comment (dmlaziuk) [inline src/pages/Restaurants.jsx]: Add an `id` field to cardsData and use it here
```suggestion
      dataSource={cardsData}
      rowKey="id"
      renderItem={(item) => (
```

File: src/pages/Restaurants.jsx [hunk 2]
```diff
@@ -1,7 +1,96 @@
...
+        <List.Item>
+          <Card
+            hoverable
+            style={{ width: 330 }}
```
Comment (dmlaziuk) [inline src/pages/Restaurants.jsx]: Please use styled components to add styles.

**Review by dmlaziuk — CHANGES_REQUESTED** [review]
1. Design should be responsive.
2. Add rating (⭐️) to the card.

File: src/pages/Restaurants.jsx (hunk 1 above)
Comment (adam-lzk) [reply src/pages/Restaurants.jsx]: Fixed

File: src/pages/Restaurants.jsx (hunk 2 above)
Comment (adam-lzk) [reply src/pages/Restaurants.jsx]: Fixed

File: src/pages/Restaurants.jsx (hunk 1 above)
Comment (dmlaziuk) [reply src/pages/Restaurants.jsx]: Not fixed.
There is no `id` field in cardsData.

Comment (adam-lzk) [reply src/pages/Restaurants.jsx]: Fixed

**Review by dmlaziuk — APPROVED** [review]

**Pull request merged**